app/views/layouts/_user_menu.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-md-3 mu-tab-body mu-user-menu">
+  <div>
+    <div class="mu-user-menu-section">
+      <div class="mu-user-menu-header">
+        <%= t(:my_account) %>
+      </div>
+      <div class="mu-user-menu-item">
+        <%= profile_user_menu_link %>
+      </div>
+    </div>
+    <div class="mu-user-menu-section">
+      <div class="mu-user-menu-header">
+        <%= t(:messages) %>
+      </div>
+      <div class="mu-user-menu-item">
+        <%= messages_user_menu_link %>
+      </div>
+      <% if should_show_discussions? %>
+        <div class="mu-user-menu-item">
+          <%= discussions_user_menu_link %>
+        </div>
+      <% end %>
```

Comment:
I really, really tried to test this and see whether 'Discussions' appeared as text or not in the page, but I couldn't get the organization to keep switched :laughing: . It'd always stop being the current one after `visit "/user"`.

Reply:
We could take a look a it together :smile:

Reply:
I'll try again for the second part of this feature